openmdao_book/other_useful_docs/api_translation.ipynb

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
<p>All the sections from the existing docs seem to have been copied</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>

Reply:
<p>They were but I just added the prob and set up so the block would run</p>